data/crac-file/crac-api/src/test/java/com/farao_community/farao/data/crac_api/NetworkElementTest.java

```diff
@@ -25,4 +25,42 @@ public void testConstructorElementTest() {
         assertEquals("basicElemId", networkElement.toString());
     }
 
+    @Test
+    public void testDifferent() {
+        NetworkElement networkElement1 = new NetworkElement("network-element-1", "name-1");
+        NetworkElement networkElement2 = new NetworkElement("network-element-2", "name-2");
+
+        assertNotEquals(networkElement1, networkElement2);
+    }
+
+    @Test
+    public void testEqualWithDifferentNames() {
+        NetworkElement networkElement1 = new NetworkElement("network-element-1", "name-1");
+        NetworkElement networkElement2 = new NetworkElement("network-element-1", "name-2");
+
+        assertEquals(networkElement1, networkElement2);
+    }
+
+    @Test
+    public void testEqualWithSameNames() {
+        NetworkElement networkElement1 = new NetworkElement("network-element-1", "name-1");
+        NetworkElement networkElement2 = new NetworkElement("network-element-1", "name-2");
+
+        assertEquals(networkElement1, networkElement2);
+    }
```

Comment:
This is exactly the same test than just above. change "name-2" in "name-1"

Reply:
Ok